src/midiEditorCommands.h

```diff
@@ -10,9 +10,10 @@
 
 const int MIDI_EDITOR_SECTION = 32060;
 
-// This should be called when playback starts, as otherwise, pending note off
+// This can be called when playback starts, as otherwise, pending note off
```

Comment:
I think this comment needs further tweaking. Calling this when playback starts is not optional; it is still a "should" at least, perhaps even a "must", definitely more than a "can". Perhaps it should be described more in terms of what it does and then how it is used.

> Cancel MIDI preview. If notesOff is true, note off messages are sent immediately. This is used to silence the preview. If notesOff is false, the pending note off messages are never sent. This is used when playback starts, as otherwise, pending note off messages for OSARA MIDI preview might interfere with REAPER MIDI playback.